### PR TITLE
Updated documentation on permissions to include FE setup

### PR DESCRIPTION
**Pull request by marijnritense**

### CODE REVIEW

**Review by Cees2505 — APPROVED** [review]

File: using-valtimo/access-control/configuring-roles.md [hunk 1]
```diff
@@ -0,0 +1,47 @@
...
+## Editing a role
+Roles can be edited by clicking on that role. Then, by clicking on the three dots next to the `Save` button, the role
+can then be updated.
+
+![updating-a-role-example](img/updating-a-role.png)
+
+## Deleting a role
+Roles can be deleted by clicking on that role. Then, by clicking on the three dots next to the `Save` button, the role
+can then be deleted.
+
```
Comment (theo-ritense) [inline using-valtimo/access-control/configuring-roles.md:25]: Maybe explain what happens to permissions when deleting a role?

EDIT:
Oh wait, you have that in a paragraph below. Can you combine these?

File: reference/modules/authorization.md [hunk 2]
```diff
@@ -65,17 +65,56 @@ No relations are supported for this type
...
+
+
+### Expression
+
+| key        | Description                                       | Examples                    |
+|:-----------|:--------------------------------------------------|:----------------------------|
+| `type`     | The type of condition. In this case `expression`. | `expression`                |
+| `field`    | The field used to start the path from.            | `content.content`           |
+| `path`     | The path that points to the field to compare to.  | `$.height`                  |
+| `operator` | The operator for the comparison.                  | `==`, `<`                   |
```
Comment (theo-ritense) [inline reference/modules/authorization.md:109]: Do we have the possible operators documented as well?

File: using-valtimo/access-control/configuring-roles.md (hunk 1 above)
Comment (marijnritense) [reply using-valtimo/access-control/configuring-roles.md:26]: done

File: reference/modules/authorization.md (hunk 2 above)
Comment (marijnritense) [reply reference/modules/authorization.md:119]: Now we do

File: using-valtimo/access-control/img/deleting-a-role.png
Comment (theo-ritense) [inline using-valtimo/access-control/img/deleting-a-role.png:1]: Wasn't this also changed to `"action": "view_list"`?

File: using-valtimo/access-control/img/updating-a-role.png
Comment (theo-ritense) [inline using-valtimo/access-control/img/updating-a-role.png:1]: Wasn't this also changed to "action": "..."?

File: using-valtimo/access-control/img/deleting-a-role.png
Comment (marijnritense) [reply using-valtimo/access-control/img/deleting-a-role.png:1]: updated

File: using-valtimo/access-control/img/updating-a-role.png
Comment (marijnritense) [reply using-valtimo/access-control/img/updating-a-role.png:1]: updated

**Review by theo-ritense — APPROVED** [review]

**Pull request merged**